### ISSUE TITLE
The unbound folder + lists are redundant

### ISSUE THREAD

**Issue by spirillen** [hagezi/dns-blocklists#3127]
You do acknowledge unbound are support the full stack of RPZ today, this means the old way for blacklisting stuff with unbound is waste of space and a redundancy of the RPZ lists you provide.

```
server:
    module-config: "respip validator iterator"
rpz:
    # The name of the RPZ authority zone
    name: hagezi.anti.piracy

    # The filename where the zone is stored. If left empty
    zonefile: hagezi.anti.piracy

    # The location of the remote RPZ zonefile.
    url: https://raw.githubusercontent.com/hagezi/dns-blocklists/main/rpz/anti.piracy.txt

    # Always use this RPZ action for matching triggers from this zone.
    # Possible action are: nxdomain, nodata, passthru, drop, disabled,
    # and cname.
    # rpz-action-override: nxdomain

    # Log all applied RPZ actions for this RPZ zone. Default is no.
    # rpz-log: yes

    # Specify a string to be part of the log line.
    # rpz-log-name: hagezi_anti_piracy
```

https://unbound.docs.nlnetlabs.nl/en/latest/topics/filtering/rpz.html

**Comment by hagezi** [comment]
Thank you, I realise that, but thank you for reminding me. The fact that it is still offered is historical. There were a few people with old Unbound versions on old RPis. There were problems with the RPZ lists. Do you know from which Unbound version RPZ is fully supported?

Are there any readers here who still use the Unbound format?

**Comment by spirillen** [comment]
> There were a few people with old Unbound versions on old RPis

Had something like that in mind when I wrote the OP, however it is years ago they released version 1.14, which was the first edition to partly support RPZ.

> Are there any readers here who still use the Unbound format?

Let me add this. Are you using a version before version 1.20.x you are holding a potential DNS bomb that needs to be upgraded sooner before later.

**Comment by hagezi** [comment]
removed.
